contracts/ocr2/src/state.rs

```diff
@@ -87,13 +87,22 @@ pub struct Config {
     pub validator: Option<Validator>,
 } // TODO: group some of these into sub-structs
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct ProposedConfig {
```

Comment:
I'll rename `Config` from the names and just use `Proposal`

Reply:
Yup, something like this could be an improvement: 

- `ExecuteMsg::BeginProposal`
- `ExecuteMsg::ClearProposal`
- `ExecuteMsg::FinalizeProposal`
- `ExecuteMsg::ApplyProposal`
- `ExecuteMsg::ProposeOracles`
- `ExecuteMsg::ProposeOffchainConfig`